stable/contour/README.md

```diff
@@ -0,0 +1,101 @@
+# Contour
+
+[Contour](https://github.com/heptio/contour) is an Ingress controller for Kubernetes that works by deploying the Envoy proxy as a reverse proxy and load balancer. Unlike other Ingress controllers, Contour supports dynamic configuration updates out of the box while maintaining a lightweight profile.
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release .
```

Comment:
This install command assumes it is a git checkout. Can you please update this to use the stable repository instead of `.`. That is for all places the commands are used.

Reply:
Fixed the Readme.